python/lsst/meas/algorithms/loadReferenceObjects.py

```diff
@@ -306,15 +306,15 @@ def makeMinimalSchema(filterNameList, addFluxSigma=False,
                 field = "%s_flux" % (filterName,),
                 type = numpy.float64,
                 doc = "flux in filter %s" % (filterName,),
-                units = "?",
+                units = "count",
             )
         if addFluxSigma:
             for filterName in filterNameList:
                 schema.addField(
                     field = "%s_fluxSigma" % (filterName,),
                     type = numpy.float64,
                     doc = "flux uncertainty in filter %s" % (filterName,),
-                    units = "?",
+                    units = "count",
```

Comment:
I don't think the units in reference catalogs are actually counts.  We had a long debate on what they should be, but I don't remember the outcome; maybe @r-owen does?